Source/Router.swift

```diff
@@ -72,11 +73,11 @@ public struct Response: ResponseFieldsProvider {
     public let headerFields: [String : String]?
     
     /**
-     Initialize `Response` object that wraps another `Serializable` object for the serialization but, implemententing `ResponseFieldsProvider` can affect some parameters of the HTTP response
+     Initialize `Response` object that wraps another `Serializable` object for the serialization but, implementing `ResponseFieldsProvider` can affect some parameters of the HTTP response
      
-     - parameter statusCode:   the status code that the response should provide to the HTTP repsonse
+     - parameter statusCode:   the status code that the response should provide to the HTTP response
      - parameter body:         the body that will be serialized
-     - parameter headerFields: the headerFields that the response should provide to the HTTP repsonse
+     - parameter headerFields: the headerFields that the response should provide to the HTTP response
```

Comment:
Ty! I'm really bad with typos I will implement Danger that apparently has typo checks!